pkgs/state-transition/src/transition.zig

```diff
@@ -64,6 +64,7 @@ pub fn verify_signatures(signedBlock: types.SignedBeamBlock) !void {
     _ = signedBlock;
 }
 
+// TODO(gballet) check if beam block needs to be a pointer
```

Comment:
it doesn't need to be but zig must be memcping it so i guess pointer would be better

Reply:
let's do the optimization in another PR, right now it works